pages/installation/virtual_machine_appliances.rst

```diff
@@ -35,6 +35,11 @@ completed:
 
 .. image:: /images/virtualbox3.png
 
+
+Note:
+If you don't have a working DHCP server for your virtual machine, you will get the error message "Your appliance came up without a configured IP address. Graylog is probable not running correctly!". 
+In this case, you have to login and edit /etc/network/interfaces in order to setup a fixed IP address. Then manually reconfigure graylog as showed in the following paragraphs.
```

Comment:
Please highlight the file name accordingly: http://docutils.sourceforge.net/docs/ref/rst/roles.html#literal